docs/usage/shoot_high_availability_best_practices.md

```diff
@@ -46,7 +46,7 @@ spec:
     workers:
     - name: ...
       minimum: 6
-      maximum: 60
+      maximum: 600
```

Comment:
`60` may be a low number, but `600` nodes is certainly not small for a cluster. I am unsure if we should recommend to plan for big clusters.
WDYT?

Reply:
Sure, I reverted this as well.